feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/SessionsFragment.kt

```diff
@@ -92,6 +93,10 @@ class SessionsFragment : DaggerFragment() {
         binding.sessionsSheet.doOnApplyWindowInsets { _, insets, _ ->
             sessionSheetBehavior.peekHeight = insets.systemWindowInsetBottom + initialPeekHeight
         }
+        binding.fragmentSessionsScrollView.doOnApplyWindowInsets { scrollView, insets, _ ->
+            // Set a bottom padding due to the system UI is enabled.
+            scrollView.updatePadding(bottom = insets.systemWindowInsetBottom + initialPeekHeight)
```

Comment:
Almost LGTM! A little thing. Can you apply a suggestion? When we add padding to ScrollView we can use it. 🥈 

```suggestion
            scrollView.updatePadding(bottom = insets.systemWindowInsetBottom + initialPeekHeight + initialState.paddings.bottom)
```

Reply:
I approved your suggestion.
but + initialState.paddings.bottom is 0dp. Are you alright?

code
```
binding.fragmentSessionsScrollView.doOnApplyWindowInsets { scrollView, insets, initialState ->
            // Set a bottom padding due to the system UI is enabled.
            Timber.debug { "scrollView insets.systemWindowInsetBottom = ${insets.systemWindowInsetBottom}" }
            Timber.debug { "scrollView initialPeekHeight = $initialPeekHeight" }
            Timber.debug { "scrollView initialState.paddings.bottom = ${initialState.paddings.bottom}" }
            scrollView.updatePadding(bottom = insets.systemWindowInsetBottom + initialPeekHeight + initialState.paddings.bottom)
        }
```

log
```
scrollView insets.systemWindowInsetBottom = 144
2020-01-17 23:09:28.232 22525-22525/io.github.droidkaigi.confsched2020.debug D/App: scrollView initialPeekHeight = 192
2020-01-17 23:09:28.232 22525-22525/io.github.droidkaigi.confsched2020.debug D/App: scrollView initialState.paddings.bottom = 0
2020-01-17 23:09:28.234 22525-22525/io.github.droidkaigi.confsched2020.debug D/App: scrollView insets.systemWindowInsetBottom = 144
2020-01-17 23:09:28.234 22525-22525/io.github.droidkaigi.confsched2020.debug D/App: scrollView initialPeekHeight = 192
2020-01-17 23:09:28.234 22525-22525/io.github.droidkaigi.confsched2020.debug D/App: scrollView initialState.paddings.bottom = 0
```

Reply:
I guess there is a controversy, but I think it will suffer when padding is added to the layout.

Reply:
I do not know the details, but once I understood. I fix code.
thanks your proposal, I deepened my understanding of code.